UI/src/i18n.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+/* global __SUPPORTED_LOCALES */
+
+export const SUPPORT_LOCALES = __SUPPORTED_LOCALES;
+
+import { nextTick } from "vue";
+import { createI18n } from "vue-i18n";
+
+const i18n = createI18n({
+    globalInjection: true,
+    legacy: false,
+    fallbackWarn: false,
+    missingWarn: false, // warning off
+    locale: window.lsmbConfig.language,
+    fallbackLocale: "en",
+    messages: {}
+});
+
+function setI18nLanguage(locale) {
+    // Update document
+    document.querySelector("html").setAttribute("lang", locale);
+    document.querySelector("html").setAttribute("xml:lang", locale);
+}
+
+export async function loadLocaleMessages(locale) {
+    if (SUPPORT_LOCALES.includes(locale)) {
+        // load locale messages
+        if (!i18n.global.availableLocales.includes(locale)) {
+            // load locale messages with dynamic import
+            const messages = await import(
+                /* webpackChunkName: "locale-[request]" */ `./locales/${locale}.json`
+            );
+
+            // set locale and locale message
+            i18n.global.setLocaleMessage(locale, messages);
+            i18n.global.locale = locale;
+        }
+        setI18nLanguage(locale);
+    }
+    return nextTick();
```

Comment:
What does this `nextTick()` call do? Can't we just `return;`? I mean, I thought `nextTick()` takes a function argument which will be called on the next tick, but without an argument, it seems pointless?

Reply:
Left over. Removed.